### PR TITLE
Update blame help

### PR DESCRIPTION
**Pull request by nohwnd**
## Summary

Updated help for dotnet test --blame and the related options.

### CODE REVIEW

File: docs/core/tools/dotnet-test.md [hunk 1]
```diff
@@ -79,6 +83,8 @@ Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework.
 - **`--blame`**
 
   Runs the tests in blame mode. This option is helpful in isolating problematic tests that cause the test host to crash. When a crash is detected, it creates a sequence file in `TestResults/<Guid>/<Guid>_Sequence.xml` that captures the order of tests that were run before the crash.
+  
+  This option does not create a memory dump, and is not helpful when the test is hanging.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:87]:
```suggestion
  This option does not create a memory dump and is not helpful when the test is hanging.
```

File: docs/core/tools/dotnet-test.md [hunk 2]
```diff
@@ -115,7 +121,7 @@ Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework.
   - 5400s, 5400sec, 5400second, 5400seconds
   - 5400000ms, 5400000mil, 5400000millisecond, 5400000milliseconds
 
-  When no unit is used (for example, 5400000), the value is assumed to be in milliseconds. When used together with data driven tests, the timeout behavior depends on the test adapter used. For xUnit and NUnit the timeout is renewed after every test case. For MSTest, the timeout is used for all test cases. This option is supported on Windows with `netcoreapp2.1` and later, on Linux with `netcoreapp3.1` and later, and on macOS with `net5.0` or later. Implies `--blame` and `--blame-hang`.
+  When no unit is used (for example, 5400000), the value is assumed to be in milliseconds. When used together with data driven tests, the timeout behavior depends on the test adapter used. For xUnit NUnit and MSTest 2.2.4+ the timeout is renewed after every test case. For MSTest before version 2.2.4, the timeout is used for all test cases. This option is supported on Windows with `netcoreapp2.1` and later, on Linux with `netcoreapp3.1` and later, and on macOS with `net5.0` or later. Implies `--blame` and `--blame-hang`.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:124]:
```suggestion
  When no unit is used (for example, 5400000), the value is assumed to be in milliseconds. When used together with data driven tests, the timeout behavior depends on the test adapter used. For xUnit, NUnit. and MSTest 2.2.4+, the timeout is renewed after every test case. For MSTest before version 2.2.4, the timeout is used for all test cases. This option is supported on Windows with `netcoreapp2.1` and later, on Linux with `netcoreapp3.1` and later, and on macOS with `net5.0` or later. Implies `--blame` and `--blame-hang`.
```

**Review by tdykstra — APPROVED** [review]
LGTM, a couple of minor punctuation suggestions.

**Comment by nohwnd** [comment]
Committed both, thanks!

**Pull request merged**